app/Console/Services/Importer.php

```diff
@@ -423,4 +460,55 @@ protected function recalculateRanks()
     {
         /*$this->comment('--- RECALCULATING RANKS ---');*/
     }
+
+    /**
+     * Generate user's API Key and email them their new password
+     */
+    protected function setupUsers()
+    {
+        $allusers = User::all();
+        foreach($allusers as $user)
+        {
+            # Generate New User Password
+            # TODO: Increase Security?
+            $newpw = substr(md5(date('mdYhs')), 0, 10);
+
+            # Generate API Key
+            $apikey = Utils::generateApiKey();
+
+            # Update Info in DB
+            $user->password = bcrypt($newpw);
```

Comment:
If you look in the registration code, there's a `Hash::make()` or something like that which will salt. Just to make sure that the password check works properly. That might use bcrypt under the hood but let's still call that one to make sure we're 100% compat